app/src/main/res/values-en/strings.xml

```diff
@@ -83,7 +83,7 @@ The info you enter is then directly added to the OpenStreetMap in your name, wit
 
     <string name="quest_generic_answer_noSign">There is no sign</string>
     <!-- e.g. "it is displayed as a footway on the map but actually these are steps" -->
-    <string name="quest_generic_answer_is_actually_steps">"This part is steps"</string>
+    <string name="quest_generic_answer_is_actually_steps">"There are steps here"</string>
```

Comment:
I think the part bit helps, because it's this specific bit (and all of it) is steps, not just some of it.

Reply:
Revised!